core/templates/dev/head/components/state-editor/state-editor-properties-services/state-editor.service.ts

```diff
@@ -36,6 +36,7 @@ oppia.factory('StateEditorService', [
     var interaction = null;
     var misconceptions = [];
     var explorationIsWhitelisted = false;
+    var solicitAnswerDetails = false;
```

Comment:
Should we start this with null and expect getter to set it correctly? (similar to correctnessFeedbackEnabled and inQuestionMode on line 30-31)

Reply:
Done